src/content/api/merchant-configs.mdx

```diff
@@ -69,6 +69,10 @@ A Merchant Config defines the available payment options for paying a [Payment Re
   <Property name="externalMerchantId" type="string">
     Id of the external merchant. Required for `epay`, `stadius`, `quartz`, and `uplinkapi.test` types.
   </Property>
+
+  <Property name="collectionIds" type="string">
+    Id of the [token collection](/api/tokens#token-collection-model), can be `discount` collection type. Required for `centrapay.ledger` types.
+  </Property>
 </Properties>
```

Comment:
@Ray2126  🙋🏻‍♀️ **Question for line  74**  - **Capitalise first letter ?**
In line 106 ,  " Only [Merchants](/api/merchants) with the `test` ....", the first letter of the word _Merchant_ is capitalised. Do we follow the pattern and update token collection as Token Collection ?

Reply:
Good catch, just updated Token Collection to be capitalized